### DISCUSSION TITLE
[Bug Report] Have to scan QR Code everytime WALC is started

### DISCUSSION THREAD

**Discussion by LaughingRover** [cstayyab/WALC#53] [Troubleshooting]
**Feathub Feature Link:** Please request this feature at https://feathub.com/cstayyab/WALC too. This issue will be used to track this feature only. Requests that are submitted at given URL will be entertained first according to number of votes on that feature.

**Is your feature request related to a problem? Please describe.**
Having to scan my whatsapp bar code everytime the app restarts tiresome

**Describe the solution you'd like**
It should have some sort of persistency so that whether the app or system restarts the whatsapp remembers your previous session until it expires or you decide to logout.

**Describe alternatives you've considered**
This is more of a problem that I think will occur when persistency is implemented. In a case of two different whatsapp running in separate windows at the same time, and the app is restarted, how do you get back those two sessions to continue running. One solution I thought of was the app itself to support multiple whatsapp

**Additional context**

---
**To view all feature requests and to vote for your favorite ones visit https://feathub.com/cstayyab/WALC This will help in determining which feature needs to be implemented first**

**Comment by daffamumtaz2361** [comment]
This isn't a feature request, this is a bug report. How did you install WALC? Snap or AppImage? What's your OS? Are you using multiple instance/window of WALC?

**Comment by tomas-zemres** [comment]
I have same Issue, I must pair with my phone via QR very often.

About says:
 WALC 0.1.11
 Installation Type: Snap
 Snap Version:0.1.11(17)OS: Ubuntu 20.04.1 LTS

I try it now:
 I pair with my phone via QR code, It works.
 I close and reopen WALC app and I see QR pairing code again.
